samples/QmlArExample/qml/main.qml

```diff
@@ -39,7 +41,44 @@ ApplicationWindow {
         id: sceneView
         anchors.fill: parent
         scene: sceneLoader.item ? sceneLoader.item.scene : null
-        onMousePressed: arcGISArView.setInitialTransformation(mouse.x, mouse.y); // for touch screen events
+
+        // If m_screenToLocationMode is true, create and place a 3D sphere in the scene. Otherwise set the
+        // initial transformation based on the screen position.
+        onMousePressed: {
+            // If "screenToLocation" mode is enabled.
+            if (screenToLocationMode)
+            {
```

Comment:
Brace on previous line for consistent Qml style. This comment applies to many places in this file.